spec/features/sign_in/multiple_vot_spec.rb

```diff
@@ -9,115 +10,278 @@
     allow(IdentityConfig.store).to receive(:doc_auth_selfie_capture_enabled).and_return(true)
   end
 
-  context 'biometric and non-biometric proofing is acceptable' do
-    scenario 'identity proofing is not required if user is proofed with biometric' do
-      user = create(:user, :proofed_with_selfie)
+  context 'with OIDC' do
```

Comment:
I recommend hiding whitespace to review this file:

![image](https://github.com/18F/identity-idp/assets/963654/1fd9eae6-be61-43f7-be55-79277203d67d)